tests/tools/TriMesh_examples.hpp

```diff
@@ -88,4 +88,11 @@ TriMesh hex_plus_two();
 //   \ / \ / \ /
 //    7---8---9
 TriMesh edge_region();
+
+//  0---2  3---4
+//  | /    | \ |
+//  1      5---6
+TriMesh three_triangles_with_two_components();
+
```

Comment:
what's the topology for nine with a hole? would be nice to be consistent about having figures for everything

Reply:
\+

Reply:
Uh, could we have a simpler picture? I think Daniel had a new test with a figure that could be cut down for this purpose